src/superphot_plus/data_generation/make_fake_spp_data.py

```diff
@@ -155,3 +180,66 @@ def create_clean_models(nmodels, num_times=100):
         lcs.append(np.array([tdata, f_model, edata, bdata]))
 
     return params, lcs
+
+
+def create_ztf_model(plot=False):
+    """
+    Generate realisitic-ish ZTF light curves from the superphot+ prior
+
+    inputs:
```

Comment:
Same formatting request as above: For consistency we use the same formatting we have been using in other doc strings?